src/Http/Middleware/CsrfProtectionMiddleware.php

```diff
@@ -64,6 +67,7 @@ class CsrfProtectionMiddleware implements MiddlewareInterface
         'expiry' => 0,
         'secure' => false,
         'httpOnly' => false,
+        'samesite' => null,
```

Comment:
I wonder if the config should be `sameSite`, camelBacked same as `httpOnly`.

Reply:
It's `httpOnly` that should have been lowercased to match the casing using in `Cookie` methods.

Reply:
The internal config string doesnt necessarily have to match the actual cookie value.
If httpOnly is like this and to avoid inconsistency, maybe we could keep camelBacked.

Reply:
I am not in favor of adding more inconsistency just to match a previous one. We can instead rename `httpOnly` to `httponly` and the constructor could check and copy old key to new one to maintain BC.

Reply:
The alternative signature of PHP's `setcookie()` method using `$options` array also uses all lowercased keys hence I had chosen the same for all options arrays in `Cookie` class.

Reply:
Deprecating `httpOnly` and using `httponly` to increase consistency with the Cookie class seems like the right move here.

Reply:
Done